src/Link.php

```diff
@@ -74,6 +82,11 @@ public static function create(string $title, \DateTimeInterface $from, \DateTime
      */
     public static function createAllDay(string $title, \DateTimeInterface $fromDate, int $numberOfDays = 1): self
     {
+        // In cases where the from date is not UTC, make sure it's UTC, size all day events are floating and non UTC dates cause bugs in the generators
+        if ($fromDate->getTimezone() !== new \DateTimeZone('UTC')) {
```

Comment:
I'm also having issues with the "All day" and gmail.
But does this line make sense? See https://3v4l.org/FAnI5
This is I believe 1 issue but I'm still having wrong dates with gmail besides this.

Reply:
Wouldn't this make more sense?
https://gist.github.com/vever001/98cba132c56c8b864126390164899bc3

Reply:
See https://github.com/spatie/calendar-links/pull/192